python/cucim/src/cucim/core/operations/color/kernel/cuda_kernel_source.py

```diff
@@ -0,0 +1,263 @@
+# Copyright (c) 2021, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cuda_kernel_code = r'''
+extern "C" {
+__global__ void brightnessjitter_kernel(unsigned char *input_rgb, \
```

Comment:
Would it make sense to put these in C file? Might make this is easier to read/lint/test/etc. and also use directly if desired. From the Python side we could always load the file into a `str` for use as needed

Reply:
@shekhardw, any thoughts on this?

Reply:
I propose to fix this in a separate MR.